ouroboros-consensus/docs/VersioningSchemeDecision.md

```diff
@@ -0,0 +1,465 @@
+    - Announce that commit as a release of the packages that have changed since the previous release.
+      (In a monorepo, there's just the one.
+      But in a polyrepo, there may be packages that haven't changed since the previous release.)
+    - Immediately merge another fresh PR that adds the `.0` component to the versions of the just released packages.
+      So what was `A.B.C.0` before the release became either `(A+1).0.0`, `A.(B+1).0`, or `A.B.(C+1)` after the first PR and now becomes either `(A+1).0.0.0`, `A.(B+1).0.0`, or `A.B.(C+1).0` after the second PR.
+
+## Proposal Redimensional124
+
+Recall that the motivation for Proposal Redimensional dismisses the concern that people and/or tools will be confused by `A.B.C.D` possible having more features or breaking changes compared to `A.B.C`.
+Those not ready to dismiss that concern can consider the following proposal, which considers enough additional shapes of dev version to eliminate that risk.
+- Each release version is `A.B.C`.
+- Each dev version is either `A`, `A.B`, or `A.B.C.0`.
+    - (We have fixed `.D` at `.0`, for consistency with the other two dev version shapes, which don't have the degree of freedom.
+      This unfortunately does increase the risks of backports/rebases/cherry-picking/etc failing to bump the version.)
+    - (It may be helpful to recognize `A.B.C.0` as `A.B.C + ϵ = A.B.(C+1) - ϵ`, `A.B` as `A.B.0 - ϵ`, and `A` as `A.0.0 - ϵ`, where ϵ stands for a 3-vector ["bigger than [`0.0.0`], but smaller than all the [others]"](https://en.wikipedia.org/wiki/Greek_letters_used_in_mathematics,_science,_and_engineering).)
+- Any PR that alters a package in a way that should affect the next release version should update its version as follows.
+    - If the version was `A` before the PR, leave it.
+    - Else if the version was `A.B` before the PR:
+        - If the PR makes a breaking change, bump it to `(A+1)`.
+        - Else leave it.
+    - Else if the version was `A.B.C.0` before the PR:
+        - If the PR makes a breaking change, bump it to `(A+1)`.
+        - Else if the PR adds new functionality, bump it to `A.(B+1)`.
+        - Else leave it.
+    - Else the version must have been `A.B.C` before the PR.
+        - If the PR makes a breaking change, bump it to `(A+1)`.
+        - Else if the PR adds new functionality, bump it to `A.(B+1)`.
+        - Else bump it to `A.B.C.0`.
+- Cut a new release as follows.
+    - Merge a fresh PR that doesn't alter the package but does bump dev version to the next release version, as follows.
+        - Bump `A` to `A.0.0`.
+        - Bump `A.B` to `A.B.0`.
+        - Bump `A.B.C.0` to `A.B.(C+1)`.
+    - Announce that commit as a release of the packages that have changed since the previous release.
+      (In a monorepo, there's just the one.
+      But in a polyrepo, there may be packages that haven't changed since the previous release.)
+    - Announce that commit.
+
+## Proposal ScrivvyRedimensional
```

Comment:
Thanks to @michaelpj for pointing me at and @zliu41 for writing https://github.com/input-output-hk/plutus/blob/master/doc/read-the-docs-site/RELEASE.adoc

Reading through it and the scripts inspired this proposal, which I think avoids the spurious git conflicts of Proposal Redimensional.

cc @dnadales @amesgen